mariadb.html.md.erb

```diff
@@ -57,13 +57,36 @@ Cloud Foundry: [Spring Music Example](https://github.com/cloudfoundry-samples/sp
 
 ## <a id='limitations'></a> Known Limitations
 
-Our HA MariaDB offering is based on Galera, which brings some [known limitations](https://mariadb.com/kb/en/mariadb/mariadb-galera-cluster-known-limitations/) with it. Please read this article carefully.
+Our HA MariaDB offering is based on Galera, which brings some [known limitations (from mariadb.com)](https://mariadb.com/kb/en/mariadb/mariadb-galera-cluster-known-limitations/) with it. Please read this article carefully.
 
-Our Galera aaS is a shared cluster. Each customer has their own database. Put simply, the service broker executes the following statements:
+A summary of Galera limitations is on [Galera Caveats](http://mysql.rjweb.org/doc.php/galera) (blockquote modified for our offering):
+
+> Dear Schema Designer:
+
+> - InnoDB only, always have PK.
+
+> Dear Developer:
+
+> - Check for errors, even after `COMMIT`.
+> - Moderate sized transactions.
+> - Don't make assumptions about `AUTO_INCREMENT` values other than uniqueness. Not monotonicity, not gapless. (This applies for non-Galera, but probably happens more often here.)
+- Handling of "critical reads" is quite different (arguably better).
+
+> Dear DBA:
+
+> - `ALTER`s are handled differently.
+> - `TRIGGER`s and `EVENT`s may need checking.
+> - Several `VARIABLE`s need to be set differently.
+
+Please add a **primary key** to every table you create. This is because of *Row Based Replication*.
+
+Please contact Swisscom Application Cloud support when you need to issue **long running DDL changes** (for example `ALTER TABLE`). Long running DDL causes an outage for all customers on the cluster.
+
+Our Galera aaS is a **shared cluster**. Each customer has their own database. Put simply, the service broker executes the following statements:
 
 ```sql
 CREATE DATABSE CF_DATABASE_NAME
-GRANT ALL PRIVILEGES ON `CF_DATABASE_NAME`.* TO 'USER'@'SUBNET'
+GRANT SELECT, INSERT, UPDATE, DELETE, CREATE, DROP, REFERENCES, INDEX, ALTER, CREATE TEMPORARY TABLES, EXECUTE, CREATE VIEW, SHOW VIEW, CREATE ROUTINE, ALTER ROUTINE, EVENT, TRIGGER ON `CF_DATABASE_NAME`.* TO 'USER'@'SUBNET'
```

Comment:
We don't really do this, but do a `GRANT ALL` followed by a `REVOKE LOCK TABLES` instead.